index.js

```diff
@@ -0,0 +1,198 @@
+  <button type="button" class="btn-close">
+    <img class="closePng" src="./closeButton.png" onclick=hideModal()>
+  </button>
+  <button type="button" class="btn-close-details">
+       <img class="DesktopPopUp" src="./SnapshootPortfolio.png" alt="cancel icon">
+   </button>
+  <div class="card-image-details-mobile" onclick=hideModal()>
+      <img src=${projectData.imagen} alt="">
+  </div>  
+  <section class="myRecentWorkPopUp"  id = "projects-card">
+      <h2 class="subTitlePopUp">${projectData.titulo}</h2> 
+      <div class="informationCard">
+          <div  class = "cardTag"></div>
+              <ul>
+                  ${projectData.tecnologias.map((tec) => `<li class="myRecentWorkItem">${tec}</li>`).join('\n')}
+              </ul>
+          </div>
+      </div>
+      <p class="card-paragraph-mobile">
+           Lorem Ipsum is simply dummy text of the printing and typesetting industrys standard dummy text ever since the 1500s
+           Lorem Ipsum is simply dummy text of the printing and typesetting industrys standard dummy text ever since the 1500s
+           Lorem Ipsum  Lorem Ipsum is simply 
+      </p>
+      <div class="card-button-mobile">
+          <a href="${projectData.btnLiveHref}" class="btn-card-details-mobile">
+          <img src="./arrowBtnComplete.svg" alt="see live icon">
+          </a>
+          <a href="${projectData.btnSourceHref}" class="btn-card-details-mobile">
+          <img src="./githubBtnComplete.svg" alt="github icon">
+          </a>
+      </div>
+  </section>
+</article>            
+  `;
+  document.querySelector('.modal').innerHTML = modalPopUp;
+  return modalPopUp;
+}
+console.log(createModal);
+// cardeElement es solo el nombre que le di para el objeto que estoy
+// creando en este caso es la tarjeta
+const createcard = (project) => {
+  // const cardModal = createModal(project)
+  // creando contenedor de la tarjeta
+  const cardElement = document.createElement('section');
+  // asignando propiedades del contenedor cardElement (class,id)
+  cardElement.classList.add('post');
+  // creando titulo de la tarjeta
+  const cardSection = document.createElement('section');
+  cardSection.classList.add('cardInformation');
+
+  const cardTitle = document.createElement('h3');
+  cardTitle.classList.add('subtitle');
+  cardTitle.innerText = project.titulo;
+  // creando tecnologias
+  const ulCard = document.createElement('ul');
+  // creando hijo de tecnologías (li)
+
+  project.tecnologias.forEach((tec) => {
+    const listItem = document.createElement('li');
+    listItem.classList.add('myRecentWorkItem');
+    listItem.innerText = tec;
+    ulCard.appendChild(listItem);
+  });
+
+  //   // creando boton
+  const buttonCardEnd = document.createElement('button');
+  buttonCardEnd.type = 'button';
+  buttonCardEnd.classList.add('buttongreen');
+  buttonCardEnd.innerText = 'See Project';
+  buttonCardEnd.setAttribute('onclick', `createModal('${project.key}')`);
+
```

Comment:
Kindly check the `See Project` button. It is not working when clicked. The button should listen to the click event according to the requirement and display the project details.
<img width="391" alt="image" src="https://user-images.githubusercontent.com/73854999/204034226-0aba69b5-d93d-44bd-9ea0-51052eefed3e.png">